docs/api.md

```diff
@@ -384,6 +397,10 @@ A valid request will return a JSON response with these properties:
 
 - `access_token`: A string that can be used for authorized requests to service providers.
 - `scope`: A string of space-separated permissions that this token has. May differ from requested scopes, since user can deny permissions.
+- `refresh_token`: (Optional) A refresh token to fetch a new access
+  token when this one expires. Only will be present if
+  `grant_type=authorization_code`.
+- `expires_at`: **Millisecond** timestamp when this access token will no longer be valid.
```

Comment:
Mixing seconds and milliseconds in this response seems like a recipe for trouble.

More broadly, I think it makes sense to copy the behaviour of existing oauth and/or openid-connect servers where there's precedent.  In this case both specs seem to suggest using `expires_in` with a relative timestamp in seconds, e.g:  http://openid.net/specs/openid-connect-core-1_0.html#ImplicitAuthResponse